spec/features/notifications/lend_request_notification_spec.rb

```diff
@@ -48,4 +48,14 @@
     expect(page).to have_text(borrower.name)
     expect(page).to have_text(item.name)
   end
+
+  it "borrower gets notified when the request is accepted" do
+    visit notifications_path
+    click_on('Lend Request')
+    click_button('Accept')
+    sign_in borrower
+    visit notifications_path
+    expect(page).to have_text(user.name)
```

Comment:
Maybe test for more than just the correct item and user as those are present in multiple notifications not just this specific one.
Consider testing for "Lending Accepted".